statsmodels/regression/tests/test_recursive_ls.py

```diff
@@ -160,7 +160,7 @@ def test_glm(constraints=None):
 
     # Regression coefficients, standard errors, and estimated scale
     assert_allclose(res.params, res_glm.params)
-    assert_allclose(res.bse, res_glm.bse)
+    assert_allclose(res.bse, res_glm.bse, atol=1e-6)
```

Comment:
Is this not passing at the moment?

Reply:
This PR adds new testing environments using PyPI packages which use different compilers and blas.